use-cases/using-SRIOV-and-kata.md

```diff
@@ -0,0 +1,325 @@
+  order to disable `CONFIG_VFIO_NOIOMMU` in the kernel configuration.
+- Optionally, you might need to add a PCI override for your Network Interface
+  Controller (NIC). The section [Checking your NIC for SR-IOV](#checking-your-nic-for-sr-iov) describes how to assess if you need to make NIC changes and how to make
+  the necessary changes.
+
+### Checking your NIC for SR-IOV
+
+The following is an example of how to use `lspci` to check if your NIC supports
+SR-IOV.
+
+```bash
+$ lspci | fgrep -i ethernet
+01:00.0 Ethernet controller: Intel Corporation Ethernet Controller 10-Gigabit X540-AT2 (rev 03)
+
+...
+$ #sudo required below to read the card capabilities
+
+$ sudo lspci -s 01:00.0 -v
+01:00.0 Ethernet controller: Intel Corporation Ethernet Controller 10-Gigabit X540-AT2 (rev 03)
+        DeviceName: NIC1
+        Subsystem: Dell Ethernet 10G 4P X540/I350 rNDC
+        Flags: bus master, fast devsel, latency 0, IRQ 146
+        Memory at 91c00000 (64-bit, prefetchable) [size=2M]
+        Memory at 91e04000 (64-bit, prefetchable) [size=16K]
+        Expansion ROM at 92800000 [disabled] [size=512K]
+        Capabilities: [40] Power Management version 3
+        Capabilities: [50] MSI: Enable- Count=1/1 Maskable+ 64bit+
+        Capabilities: [70] MSI-X: Enable+ Count=64 Masked-
+        Capabilities: [a0] Express Endpoint, MSI 00
+        Capabilities: [e0] Vital Product Data
+        Capabilities: [100] Advanced Error Reporting
+        Capabilities: [150] Alternative Routing-ID Interpretation (ARI)
+        Capabilities: [160] Single Root I/O Virtualization (SR-IOV)
+        Capabilities: [1d0] Access Control Services
+        Kernel driver in use: ixgbe
+        Kernel modules: ixgbe
+```
+
+The important line being:
+
+```
+        Capabilities: [160] Single Root I/O Virtualization (SR-IOV)
+```
+
+If your card does not report this capability, then it does not support SR-IOV.
+
+### IOMMU Groups and PCIe Access Control Services
+
+Run the following command to see how the IOMMU groups are setup on your
+host system.
+```
+$ find /sys/kernel/iommu_groups/ -type l
+```
+
+The command's output details whether or not your NIC is setup
+appropriately with respect to PCIe Access Control Services (ACS).
+If the IOMMU groups are setup properly, the PCI for each ACS-enabled NIC port
+should be in its own IOMMU group. If the PCI bridge is within the same IOMMU
+group as your NIC, it indicates that either your device does not support ACS
+or your device does not appropriately share this default capability.
+
+If you do not see any output when running the previous
+command, then you likely need to update your host's kernel configuration.
+
+For more details, see http://vfio.blogspot.com/2014/08/iommu-groups-inside-and-out.html
+
+### Update the host kernel
+
+
+Depending on your host kernel configuration, you might have to rebuild the
+kernel. If the following conditions are true, you do not need to rebuild
+your kernel:
```

Comment:
I'd add a warning about this since it is less than ideal that users have to rebuild their host kernel:

> **WARNINGS:**
>
> - You need to be *very clear* of the security and maintenance implications of creating a new **host kernel**.
> - Mistakes in installing new kernels and updating the bootloader could make your system unbootable.
> - We advise you to ensure you have a recent (and tested) full system backup before proceeding.